docs/guides/admin/docs/configuration/stream-security.md

```diff
@@ -220,6 +220,25 @@ Example:
     url.regex.archive=.*archive\/archive\/mediapackage\/.*\/.*\/.*
     url.regex.static=.*static.*
 
+Configuration of Wowza URL Signing
```

Comment:
So if I understand this correctly, this is not Stream Security, but a different way of signing URLs specifically for Wowza? (Cause I'm pretty sure the first already works.) That's bound to cause some confusion...